chart/templates/builtin-registry-certs-secret.yaml

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License. See License-MIT.txt in the project root for license information.
 
 {{ if (index .Values "docker-registry" "enabled") }}
-{{- $regName := include "gitpod.builtinRegistry.name" . -}}
+{{- $regName := include "gitpod.builtinRegistry.internal_name" . -}}
 {{ $cm := (index .Values "cert-manager") }}
 {{- if $cm.enabled }}
```

Comment:
@wulfthimm: The (self-signed) cert of the registry has always the internal domain `registry.default.svc.cluster.local` as domain, never the public `registry.my-gitpod.example.com` domain. Don't know how the cert-manager works but could this lead to problems with the cert-manager configuration here? Do we need to drop this?